lib/gssapi/krb5/acquire_cred.c

```diff
@@ -33,6 +33,42 @@
 
 #include "gsskrb5_locl.h"
 
+/*
+ * Find an element in a cred store. Returns GSS_S_COMPLETE if the cred store
+ * is absent or well formed, irrespective of whether the element exists.
+ */
+OM_uint32
+__gsskrb5_cred_store_find(OM_uint32 *minor_status,
+			  gss_const_key_value_set_t cred_store,
+			  const char *key,
+			  const char **value,
+			  int internal)
```

Comment:
`internal` is no longer needed since you removed `__gss_cred_store_password_key`.